### PR TITLE
Add Keyboard Shortcuts For Pausing Game & Not Requeuing Next Match

### PR DESCRIPTION
**Pull request by Kyrluckechuck**
## Description
This re-adds the keyboard shortcuts functionality from https://github.com/Kyrluckechuck/TFT-Bot, adding initially just two functions:
- To pause the bot so that the "next" action won't occur
- To stop the bot from re-queing next game, which is useful for if you want it to finish up the game but not start another so you can use it, etc.

I'm open to suggestions on cleaner implementations, but basically just ripped it right out of the old bot for simplicity 😅

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -293,6 +298,12 @@ async def loop(adb_instance: ADB, config: AluneConfig):
         config: An instance of the alune config to use.
     """
     while True:
+        delay_next_game()
+
+        if PAUSE_LOGIC:
+            time.sleep(5)
```
Comment (akshualy) [inline main.py:304]: Let's stay with asyncio.sleep in async functions :)

File: main.py [hunk 2]
```diff
@@ -332,6 +343,10 @@ async def loop(adb_instance: ADB, config: AluneConfig):
                 screenshot = await adb_instance.get_screen()
                 search_result = screen.get_button_on_screen(screenshot, Button.exit_now)
                 while not search_result:
+                    if PAUSE_LOGIC:
+                        time.sleep(5)
```
Comment (akshualy) [inline main.py:347]: here as well~

File: main.py [hunk 3]
```diff
@@ -495,6 +511,57 @@ async def main():
...
+    else:
+        logger.warning("Bot will queue a new game when in lobby!")
+
+
+def setup_hotkeys() -> None:
+    """
+    Setup hotkey listeners
+    """
+    keyboard.add_hotkey("alt+p", lambda: toggle_pause())  # pylint: disable=unnecessary-lambda
+    keyboard.add_hotkey("alt+n", lambda: toggle_play_next_game())  # pylint: disable=unnecessary-lambda
```
Comment (akshualy) [inline main.py:562]: Is this compatible with MacOS? I'm on mobile so can't look it up that well

**Review by akshualy — APPROVED** [review]
There probably can be logic improvements but I trust you that this will work :p Feel free to merge after the sleep changes

File: main.py (hunk 1 above)
Comment (Kyrluckechuck) [reply main.py]: Hadn't looked very hard and didn't notice, done!

**Pull request merged**